guides/common/modules/con_resolving-package-dependencies.adoc

```diff
@@ -46,7 +46,7 @@ ifeval::["{client-os-family}" == "Red Hat"]
 [id="Combining_exclusion_filters_with_dependency_solving_{context}"]
 .Combining exclusion filters with dependency solving
 ====
-You want to recreate {RHEL}{nbsp}8.3 using content view filters and include selected errata from a later {RHEL}{nbsp}8 minor release.
+You want to recreate {RHEL}{nbsp}8.3 by using content view filters and include selected errata from a later {RHEL}{nbsp}8 minor release.
```

Comment:
"You want" is ambiguous. 

Are you saying that the user is required to do this? "Recreate RHEL 8.3  ..." or "You must recreate RHEL 8.3..."

Or is this an option? "You can recreate RHEL 8.3 ..."

Reply:
I think it's an example like "Let's say you want to...".

Reply:
```suggestion
For example, you recreate {RHEL}{nbsp}8.3 by using content view filters and include selected errata from a later {RHEL}{nbsp}8 minor release.
```